### PR TITLE
upd(hyprland): `0.34.0` -> `0.35.0`

### PR DESCRIPTION
**Pull request by Elsie19**

### CODE REVIEW

File: packages/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland.pacscript [hunk 1]
```diff
@@ -12,9 +12,7 @@ pkgdesc="xdg-desktop-portal backend for hyprland"
 maintainer="Oren Klopfer <oren@taumoda.com>"
 arch=("amd64" "arm64")
 depends=("pipewire" "xdg-desktop-portal" "grim" "slurp")
-makedepends=("cmake" "pkg-config" "meson" "ninja-build" "scdoc" "wayland-utils" "wayland-protocols" "libgbm-dev" "libinih-dev" "uuid-dev" "qt6-base-dev" "qt6-wayland-dev" "libpipewire-0.3-dev" "libsdbus-c++-dev" "libsdbus-c++-bin")
-pacdeps=("libdrm")
-
+makedepends=("cmake" "pkg-config" "meson" "ninja-build" "scdoc" "wayland-utils" "wayland-protocols" "libgbm-dev" "libinih-dev" "uuid-dev" "qt6-base-dev" "qt6-wayland-dev" "libpipewire-0.3-dev" "libsdbus-c++-dev" "libsdbus-c++-bin" "libdrm-dev")
```
Comment (Elsie19) [inline packages/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland.pacscript:15]:
```suggestion
makedepends=("cmake" "pkg-config" "meson" "ninja-build" "scdoc" "wayland-utils" "wayland-protocols" "libgbm-dev" "libinih-dev" "uuid-dev" "qt6-base-dev" "qt6-wayland-dev" "libpipewire-0.3-dev" "libsdbus-c++-dev" "libsdbus-c++-bin" "libdrm-dev")

```

File: packages/hyprland/hyprland.pacscript [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-pacdeps=('libtomlplusplus-header' 'libdrm' 'xdg-desktop-portal-hyprland')
+makedepends=('meson' 'ninja-build' 'cmake-extras' 'cmake' 'gettext' 'gettext-base' 'libfontconfig-dev' 'libffi-dev' 'libxml2-dev' 'libxkbcommon-x11-dev' 'libxkbregistry-dev' 'libxkbcommon-dev' 'libpixman-1-dev' 'libudev-dev' 'libseat-dev' 'seatd' 'libxcb-dri3-dev' 'libvulkan-dev' 'libvulkan-volk-dev' 'libvkfft-dev' 'libgulkan-dev' 'libegl-dev' 'libgles2' 'libegl1-mesa-dev' 'glslang-tools' 'libinput-bin' 'libinput-dev' 'libxcb-composite0-dev' 'libavutil-dev' 'libavcodec-dev' 'libavformat-dev' 'libxcb-ewmh2' 'libxcb-ewmh-dev' 'libxcb-present-dev' 'libxcb-icccm4-dev' 'libxcb-render-util0-dev' 'libxcb-res0-dev' 'libxcb-xinput-dev' 'xdg-desktop-portal-wlr' 'fontconfig' 'libliftoff-dev' 'libdisplay-info-dev' 'libpango1.0-dev' 'libsystemd-dev' 'libgbm-dev' 'hwdata' 'xwayland' 'wayland-protocols' 'libwayland-dev' 'cpio' 'libtomlplusplus-dev')
+pacdeps=('xdg-desktop-portal-hyprland')
+case "${DISTRO}" in
+  ubuntu:noble | ubuntu:devel) makedepends+=('vulkan-utility-libraries-dev') ;;
```
Comment (Elsie19) [inline packages/hyprland/hyprland.pacscript:9]:
```suggestion
  ubuntu:noble | ubuntu:devel) pacdeps+=('libtomlplusplus-header') makedepends+=('vulkan-utility-libraries-dev') ;;
```

File: packages/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland.pacscript [hunk 3]
```diff
@@ -6,14 +6,13 @@ url="https://github.com/hyprwm/${name}/archive/v${pkgver}/v${pkgver}.tar.gz"
 protocols_ver="0.2"
 protocols_url="https://github.com/hyprwm/hyprland-protocols.git"
 hash="c8ba3d173a580d8b4ec9a5ee15e7780d266e536a78a12aa28d8234e0854ae86d"
-compatible=('ubuntu:devel' 'debian:sid' 'ubuntu:mantic' 'ubuntu:noble')
+compatible=('ubuntu:devel' 'debian:sid' 'ubuntu:mantic' 'ubuntu:noble' 'ubuntu:jammy')
```
Comment (Elsie19) [inline packages/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland.pacscript:9]:
```suggestion
compatible=('ubuntu:devel' 'debian:sid' 'ubuntu:mantic' 'ubuntu:noble')
```

File: packages/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland.pacscript [hunk 4]
```diff
@@ -12,8 +12,7 @@ pkgdesc="xdg-desktop-portal backend for hyprland"
 maintainer="Oren Klopfer <oren@taumoda.com>"
 arch=("amd64" "arm64")
 depends=("pipewire" "xdg-desktop-portal" "grim" "slurp")
-makedepends=("cmake" "pkg-config" "meson" "ninja-build" "scdoc" "wayland-utils" "wayland-protocols" "libgbm-dev" "libinih-dev" "uuid-dev" "qt6-base-dev" "qt6-wayland-dev" "libpipewire-0.3-dev" "libsdbus-c++-dev" "libsdbus-c++-bin")
-pacdeps=("libdrm")
+makedepends=("cmake" "pkg-config" "meson" "ninja-build" "scdoc" "wayland-utils" "wayland-protocols" "libgbm-dev" "libinih-dev" "uuid-dev" "qt6-base-dev" "qt6-wayland-dev" "libpipewire-0.3-dev" "libsdbus-c++-dev" "libsdbus-c++-bin" "libdrm-dev")
```
Comment (Elsie19) [inline packages/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland.pacscript:15]:
```suggestion
makedepends=("cmake" "pkg-config" "meson" "ninja-build" "scdoc" "wayland-utils" "wayland-protocols" "libgbm-dev" "libinih-dev" "uuid-dev" "qt6-base-dev" "qt6-wayland-dev" "libpipewire-0.3-dev" "libsdbus-c++-dev" "libsdbus-c++-bin")
case "${DISTRO}" in
  ubuntu:mantic) pacdeps+=('libdrm') ;;
  *) makedepends+=('libdrm-dev') ;;
esac
```

**Comment by Elsie19** [comment]
Stalled because `xdg-desktop-portal-hyprland` fails to build *fully* on mantic but does on noble/devel/rhino:

<details>
<summary>Picture of Mantic tree</summary>

image

</details>

<details>
<summary>Text of Rhino tree</summary>

```
rhino@docker-desktop:~$ pacstall -T xdg-desktop-portal-hyprland
/usr
  │──share
  │  │──licenses
  │  │  │──xdg-desktop-portal-hyprland
  │  │  │  │──LICENSE
  │  │──xdg-desktop-portal
  │──local
  │  │──share
  │  │  │──hyprland-protocols
  │  │  │  │──protocols
  │  │  │  │  │──hyprland-toplevel-export-v1.xml
  │  │  │  │  │──hyprland-global-shortcuts-v1.xml
  │  │  │──dbus-1
  │  │  │  │──services
  │  │  │  │  │──org.freedesktop.impl.portal.desktop.hyprland.service
  │  │  │──xdg-desktop-portal
  │  │  │  │──portals
  │  │  │  │  │──hyprland.portal
  │  │  │──pkgconfig
  │  │  │  │──hyprland-protocols.pc
  │  │──libexec
  │  │  │──xdg-desktop-portal-hyprland
  │  │──lib
  │  │  │──systemd
  │  │  │  │──user
  │  │  │  │  │──xdg-desktop-portal-hyprland.service
  │  │──bin
  │  │  │──hyprland-share-picker
  │──bin
  │  │──hyprland-share-picker
rhino@docker-desktop:~$ 
```

</details>

**Comment by oklopfer** [comment]
Try updating XDPH first, see if it builds on mantic https://github.com/hyprwm/xdg-desktop-portal-hyprland/releases/tag/v1.3.1

**Comment by Elsie19** [comment]
@oklopfer approve?

**Pull request merged**